### PR TITLE
[Security] `az security atp cosmosdb`: Add CLI support for ATP settings (Defender) on Cosmos DB

### PR DESCRIPTION
**Pull request by HaimBendanan**
**Related command**
```
az security atp cosmosdb show--resource-group MyResourceGroup --cosmosdb-account MyCosmosDbAccount 
az security atp cosmosdb update --resource-group MyResourceGroup --cosmosdb-account MyCosmosDbAccount --is-enabled 
true
```
**Description**
Adding support for get/set ATP settings (Defender) on Cosmos DB resources.
The REST api supports both Storage accounts and Cosmos DB accounts, but until now, the CLI supported only Storage accounts.
In this PR, we added support for Cosmos DB.

The REST api: 
https://docs.microsoft.com/en-us/rest/api/securitycenter/advanced-threat-protection/get
https://docs.microsoft.com/en-us/rest/api/securitycenter/advanced-threat-protection/create

**Testing Guide**
See related commands section above

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Security

**Comment by HaimBendanan** [comment]
For more context, check out this PR where we added the initial ATP support: https://github.com/Azure/azure-cli/pull/11661

**Comment by HaimBendanan** [comment]
@yonzhan @evelyn-ys @kairu-ms @jiasli @jsntcy 
Hey ping :)

File: src/azure-cli/azure/cli/command_modules/security/commands.py [hunk 1]
```diff
@@ -283,11 +283,12 @@ def load_command_table(self, _):
...
-    with self.command_group('security atp storage',
-                            security_advanced_threat_protection_sdk,
-                            client_factory=cf_security_advanced_threat_protection) as g:
-        g.custom_show_command('show', 'get_atp_setting')
-        g.custom_command('update', 'update_atp_setting')
+    for scope in ['storage', 'cosmosdb']:
+        with self.command_group(f"security atp {scope}",
+                                security_advanced_threat_protection_sdk,
+                                client_factory=cf_security_advanced_threat_protection) as g:
+            g.custom_command('show', f"get_{scope}_atp_setting")
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/security/commands.py:290]:
```suggestion
            g.custom_show_command('show', f"get_{scope}_atp_setting")
```

**Review by jiasli — APPROVED** [review]

**Pull request merged**